helpers/response/create_response.py

```diff
@@ -1,51 +1,55 @@
 from datetime import datetime
 from api.response.models import Response
 from api.room_resource.models import Resource
+from api.room.models import RoomResource
 
 
 def create_response(question_type, errors, responses, **kwargs):
-    if question_type.lower() == 'rate':
-        if 'rate' in kwargs and kwargs['rate']:
-            rating = [1, 2, 3, 4, 5]
-            if kwargs['rate'] not in rating:
-                errors.append(
-                    'Please rate between 1 and 5 for question {}'.format(
-                        kwargs['question_id']))
+    if question_type.lower() == 'rate' and 'rate' in kwargs and kwargs['rate']: # noqa
```

Comment:
I think `and 'rate' in kwargs` has the same truth value as `and kwargs['rate']`

Reply:
@mnswaleh Not really. `rate in kwargs` checks whether the key `rate` is in the dict `kwargs`.
`kwargs['rate']` checks if the value of the key `rate` is truthy